### PR TITLE
PURCHASE-1340: Adds boilerplate and initial test

### PR DESCRIPTION
**Pull request by sweir27**
I followed [Cypress's instructions](https://docs.cypress.io/guides/getting-started/installing-cypress.html) for setting up for the most part.

I also referenced this [recommended blogpost](https://basarat.gitbooks.io/typescript/docs/testing/cypress.html) for setting it up with typescript.

This includes a `Hello world` test which pulls up Artsy.net, clicks on the second link (which takes you to the /collect page). I imagine there has to be a way to select the link based on its text (vs. indexing into the list). We may want to write a helper for this.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "baseUrl": "../node_modules",
```
Comment (ds300) [inline tsconfig.json]: This looks odd and dangerous to me. `baseUrl` is used to configure absolute path resolution in tsc. This seems to be pointing it up out of the repo dir. Do you remember where this config file came from, and does it still work if you remove the `baseUrl` option?

Comment (sweir27) [reply tsconfig.json]: Oooh let me check.

Comment (sweir27) [reply tsconfig.json]: Actually something's wrong here, one sec..

**Comment by sweir27** [comment]
Will merge per our offline conversation.

**Pull request merged**